docs/about.md

```diff
@@ -17,36 +17,24 @@ RepoSense is a project based in the [National University of Singapore, School of
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Peh Xian Bin, Eugene](https://github.com/eugenepeh)
-![](https://avatars.githubusercontent.com/u/19277206?s=150&v=4)<br/>
-**Role**: Project Lead [2018 May - ]<br/>
-
-<!-- ------------------------------------------------------------------------------------------------------ -->
-
-### [Teng Yong Hao](https://github.com/yong24s)
-![](https://avatars2.githubusercontent.com/u/2003406?s=150&v=4)<br/>
-**Role**: Mentor [2018 Dec - ]<br/>
-Committer [2018 Aug - 2018 Dec]<br/>
-Contributor [2018 May - 2018 Aug]<br/>
-
-<!-- ------------------------------------------------------------------------------------------------------ -->
-
 ### [Wang Chao](https://github.com/fzdy1914)
 ![](https://avatars3.githubusercontent.com/u/35621726?s=150&v=4)<br/>
-**Role**: Committer [2019 May - ]<br/>
+**Role**: Project Lead [2021 Jan - ]<br/>
+Committer [2019 May - 2021 Jan]<br/>
```

Comment:
Hi, James, can you help me add a maintainer Phase?

Project Lead [2021 Jan - ]<br/>
Maintainer [2020 May - 2021 Jan]<br/>
Committer [2019 May - 2020 May]<br/>

You can also mention that you have been a `committer` from 2020 May.

Reply:
done